code/datums/mutations/touch.dm

```diff
@@ -29,6 +29,7 @@
 	item_state = "zapper"
 
 /obj/item/melee/touch_attack/shock/afterattack(atom/target, mob/living/carbon/user, proximity)
+	user.Beam(target, icon_state="lightning[rand(1,12)]", time=5, maxdistance = INFINITY)
```

Comment:
Is infinite max distance ever a good idea

Reply:
The max distance doesn't really matter because it'll just stop at the target

Reply:
Still never a good idea because it's just more likely by some strand of unluckiness we get an infinite loop, whether the target is deleted mid beam or god knows what could happen. It's better to put a hard cap on. Something high like 20 would be better.

Reply:
Fair enough